graphql/resolvers/auth.js

```diff
@@ -0,0 +1,47 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const User = require("../../models/user");
+
+module.exports = {
+  createUser: async (args) => {
+    try {
+      const existingUser = await User.findOne({ email: args.userInput.email });
+      if (existingUser) {
+        throw new Error("User exist already.");
+      }
+      const hashedPassword = await bcrypt.hash(args.userInput.password, 12);
```

Comment:
It's better to write a pre save method on User schema itself to avoid this writing every time. You can look at the donut-backend User schema.

Reply:
Cool, I'll look into it.